hack/tools.mk

```
@@ -175,6 +177,9 @@ $(OPENAPI_GEN): go.mod
 $(PROMTOOL): $(TOOLS_PKG_PATH)/install-promtool.sh
 	@$(TOOLS_PKG_PATH)/install-promtool.sh
 
+$(PROTOC): $(call tool_version_file,$(PROTOC),$(PROTOC_VERSION))
+	@PROTOC_VERSION=$(PROTOC_VERSION) $(TOOLS_BIN_DIR)/../../download-protoc.sh
```

Can we move the installation script and make it consistent with `install-promtool.sh`?
```suggestion
	@PROTOC_VERSION=$(PROTOC_VERSION) $(TOOLS_PKG_PATH)/install-protoc.sh
```